### PR TITLE
Update links to docs

### PR DESCRIPTION
**Pull request by endast**
The links returned 404

### CODE REVIEW

File: videointelligence/README.rst [hunk 1]
```diff
@@ -37,5 +37,5 @@ API enables developers to
 understand the content of a video by encapsulating powerful machine
 learning models in an easy to use REST API.
 
-.. _Video Intelligence: https://cloud.google.com/videointelligence/
-.. _Video Intelligence API docs: https://cloud.google.com/videointelligence/reference/rest/
+.. _Video Intelligence: https://cloud.google.com/video-intelligence/
+.. _Video Intelligence API docs: https://cloud.google.com/video-intelligence/docs/
```
Comment (dhermes) [inline videointelligence/README.rst]: @endast This should (probably) be: https://cloud.google.com/video-intelligence/docs/reference/rest/

@lukesneeringer Can you confirm?

Comment (lukesneeringer) [inline videointelligence/README.rst]: They both look valid to me.

Comment (dhermes) [inline videointelligence/README.rst]: @lukesneeringer The issue is not the validity. I was asking if the invalid link

https://cloud.google.com/videointelligence/reference/rest/

should be replaced with

1. https://cloud.google.com/video-intelligence/docs/ `OR`
1. https://cloud.google.com/video-intelligence/docs/reference/rest/

Comment (lukesneeringer) [inline videointelligence/README.rst]: Oh, I see. The latter.

Comment (dhermes) [inline videointelligence/README.rst]: @endast Can you LMK when you've updated the link, then we can merge?

**Comment by lukesneeringer** [comment]
@dhermes This should be updated. Please verify and merge if so.

**Pull request merged**